py_gtfs_rt_ingestion/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
```

Comment:
this file describes the project, its dependencies, and scripts that we'll want to run as we go.

from my understanding, it is largely edited using the `poetry` command line tool?